### ISSUE TITLE
ComboBox is not usable when using VoiceOver on Iphone 10.3

### ISSUE THREAD

**Issue by kietresolve** [telerik/kendo-angular#1000]
Hi team,

We are developing an application using angular kendo ui abd we care about accessibility. At the moment our tests are failing at the angular kendo combo box when it is used in Iphone 10.3 with VoiceOver turned on...

 Steps:

* Double tab on the combo box the popup for option menus appeared but cannot swipe into the options to select one of them...

Expect:

double tab into combobox would open iphone's selection interface.
Or popup menu would be modal where the user cannot navigate away from it until a selection is made...

Thanks in advance.

Kiet Tran

**Comment by Raisolution** [comment]
Further details and proposed solution provided in private thread 1563366.
